bundles/org.openhab.binding.metofficedatahub/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,589 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="metofficedatahub"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Channel Groups for MetOfficeDataHub Binding - Site API -->
+	<channel-group-type id="siteApiCommonDataGroup">
+		<label>Site API - Common Response Data</label>
+		<description>Site Specific Location weather forecast - Common Data</description>
+		<channels>
+			<channel id="locationName" typeId="locationNameType"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="siteApiHourlyForecastGroup">
+		<label>Site API - Hourly Forecast</label>
```

Comment:
```suggestion
		<label>Hourly Forecast</label>
```
Your not going to like this i guess, but adjust all labels in this XML-file to:
_<25 chars, max 2-3 words and properly capitalized_

Aslso run the i18n plugin when this is updated.

Reply:
I have reduced as much as possible, while attempting to keep the context - hopefully the new names are close enough